lib/components/Charts/AreaChart/index.tsx

```diff
@@ -30,6 +37,7 @@ export const _AreaChart = <
     xAxis,
     yAxis,
     lineType,
+    showLegend,
```

Comment:
Wondering if this should be optional or not. In the end, this design system is all about having opinions and reducing the amount of choices.

Reply:
Hmmm I agree, I found some examples at figma that we can use legend and some not, so I made it optional. But since the design is not final, I will always show it for now

Reply:
[always show area chart legend](https://github.com/factorialco/factorial-one/pull/355/commits/dac7b65721f0e2486892a1f1dfe55624601f7013)